easyconfigs/LAMMPS_OBMD-foss-2023b-software-commit.eb

```
@@ -0,0 +1,107 @@
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = source_urls = ['https://github.com/chocolatesoup/OBMD-LAMMPS-extension/archive/']
+sources = ['%(version)s.tar.gz']
+start_dir = 'code' # subdirectory containing the actual code
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('pkgconf', '2.0.3'),
+    ('archspec', '0.2.2'),
+    ('PyYAML', '6.0.1'),
+]
+dependencies = [
+    ('Python', '3.11.5'),
+    ('libpng', '1.6.40'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('netCDF', '4.9.2'),
+    ('GSL', '2.7'),
+    ('zlib', '1.2.13'),
+    ('gzip', '1.13'),
+    ('cURL', '8.3.0'),
+    ('HDF5', '1.14.3'),
+    ('PCRE', '8.45'),
+    ('libxml2', '2.11.5'),
+    ('FFmpeg', '6.0'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.3.0'),
+    ('Eigen', '3.4.0'),
+    ('PLUMED', '2.9.2'),
+    ('SciPy-bundle', '2023.11'),
+    # VTK package is auto-disabled if this dep is not available
+    ('VTK', '9.3.0'),
+    # We use a custom build of MDI
+    ('MDI', '1.4.29'),
+    ('mpi4py', '3.1.5'),
+]
+if ARCH == 'x86_64':
+    # TBB and ScaFaCos are an optional dependency when building on Intel arch
+    dependencies += [
+        ('tbb', '2021.13.0'),
+        ('ScaFaCoS', '1.0.4'),
+    ]
+
+# To use additional custom configuration options, use the 'configopts' easyconfig parameter
+# See docs and lammps easyblock for more information.
+# https://github.com/lammps/lammps/blob/master/cmake/README.md#lammps-configuration-options
+
+# OpenMP-Kokkos build is default in the current easyblock. One can switch to serial backend of Kokkos,
+# which is claimed to be faster in pure MPI calculations
+# configopts  = "-DKokkos_ENABLE_SERIAL=yes "
+
+
+# packages auto-enabled by easyblock
+# 'GPU'    - if cuda package is present and kokkos is disabled
+# 'KOKKOS' - if kokkos is enabled (by default)
+# 'INTEL'  - if builing on Intel CPU
+# 'OPENMP' - if OpenMP swithed on in 'toolchainopts'
+
+# include the following extra packages into the build
+general_packages = [
+    'DPD-BASIC',
+    'MOLECULE',
+    'OBMD'
+]
+
+# Excluded packages due to requiring additional (non-trivial) deps
+# - ADIOS
+# - LATTE
+# - MESONT (requires very large files downloaded during build)
+# - ML-HDNNP (requires N2P2)
+# - ML-QUIP
+# - MSCG
+# - QMMM (setup seems complex)
+
+# remove example atm because MANYBODY is not enabled
+# remove example colloid because COLLOID package is not enabled
+# remove example dipole because DIPOLE package is not enabled
+# remove example hugoniostat because EXTRA-PAIR package is not enabled
+# removing example msst bacause SHOCK package is not enabled
+# removing example pour because GRANULAR package is not enabled
+# removing example voronoi beacuse VORONOI package is not enabled
+
+
```

This one is also required for the newer easyblock
```suggestion
runtest = False
```